docs: 📚 Add documentation to all discontinued ui-components from CL (#1294)
solid-design-system/solid: packages/components/src/docs/Migration/Intro.mdx

}
```

### 🚫 Discontinued ui-components from [Component Library](https://component-library.fe.union-investment.de/latest/storybook/index.html)

Teams that have been using the following components should be aware that they will **not** be supported in the **Solid Design System (SDS) library**. If you have been relying on these components, it’s essential to implement their functionality in your own custom library.

> Note: Our practice involves detailed discussions about each UI component before migrating it into the SDS. As a result, this list will be regularly updated and adjusted.

- ui-contact: The functionality of this component is too specific to be included in the SDS.
- ui-link-list: Simimar to `ui-contact`, the functionality of this component is too specific.
- ui-iframe: This component is removed as unlike, `ui-video`, `sd-video` does not depend on it.
- ui-portal: The component adds unnecessary complexity to the library.

> - ui-superteaser: This component is replaced by `sd-video`.

Reviewer: ```suggestion
- ui-superteaser: The functionality of this component is too specific to be included in the SDS.
```

- ui-text-image: The component addresses a very specific or rare use case that doesn't justify its continued inclusion in the library.
- ui-tag-group: This component is easily replicable using the sd-tag component with custom CSS styles (e.g., flexbox).

> - ui-qr-code: The component is not widely used across projects or by the target audience, making it unnecessary to maintain.

Reviewer: ```suggestion
- ui-qr-code: The component is not widely used across projects or by the target audience, making it unnecessary to maintain within the SDS.
```

- ui-social-follow-links: The component adds unnecessary complexity to the library, both in terms of usage and maintenance.
- ui-social-share-buttons: Similar to `ui-social-follow-links`, this component adds unnecessary complexity to the library.
- ui-teaser-article: The functionality provided by the component is already covered by `sd-teaser` and `sd-media-teaser`.

> - ui-surrounding-map: This component was heavily dependent on the Google Maps API and had often changing requirements.

Reviewer: ```suggestion
- ui-surrounding-map: This component was heavily dependent on the Google Maps API and had often changing requirements. The SDS provides the `sd-map-marker` to be used inside a Google Maps map and in addition to this a guide on how to customize a Google Maps implementation by using the style wizzard.
```

Reviewer: I unfortunately was not able to finde the docs about the wizzard on Storybook or Figma. @mariohamann or @coraliefeil do you remember where we added this? 

@Vahid1919 as soon as we have the link, I would link to it within the text above.

Reviewer: :) I found it in the other PR... 

But I remember, that we had a more specific docs about how to use the wizzard in Figma?!? this would be nice to be linked in here AND maybe in the migration guide. What do you think?

Author: I totally agree with finding more specific link to docs. Waiting for input from @coraliefeil or @mariohamann :)

> - ui-contact: The functionality of this component is too specific to be included in the SDS.
> - ui-link-list: Simimar to `ui-contact`, the functionality of this component is too specific.
> - ui-iframe: This component is removed as unlike, `ui-video`, `sd-video` does not depend on it.
> - ui-portal: The component adds unnecessary complexity to the library.
> - ui-superteaser: The functionality of this component is too specific to be included in the SDS.
> - ui-text-image: The component addresses a very specific or rare use case that doesn't justify its continued inclusion in the library.
> - ui-tag-group: This component is easily replicable using the sd-tag component with custom CSS styles (e.g., flexbox).
> - ui-qr-code: The component is not widely used across projects or by the target audience, making it unnecessary to maintain within the SDS.
> - ui-social-follow-links: The component adds unnecessary complexity to the library, both in terms of usage and maintenance.
> - ui-social-share-buttons: Similar to `ui-social-follow-links`, this component adds unnecessary complexity to the library.
> - ui-teaser-article: The functionality provided by the component is already covered by `sd-teaser` and `sd-media-teaser`.
> - ui-surrounding-map: This component was heavily dependent on the Google Maps API and had often changing requirements. The SDS provides the `sd-map-marker` to be used inside a Google Maps map and in addition to this a guide on how to customize a Google Maps implementation by using the [Google Maps Style Wizard](https://mapstyle.withgoogle.com/).

Reviewer 2: ```suggestion
- `ui-contact`: The functionality of this component is too specific to be included in the SDS.
- `ui-link-list`: Simimar to `ui-contact`, the functionality of this component is too specific.
- `ui-iframe`: This component is removed as unlike, `ui-video`, `sd-video` does not depend on it.
- `ui-portal`: The component adds unnecessary complexity to the library.
- `ui-superteaser`: The functionality of this component is too specific to be included in the SDS.
- `ui-text-image`: The component addresses a very specific or rare use case that doesn't justify its continued inclusion in the library.
- `ui-tag-group`: This component is easily replicable using the sd-tag component with custom CSS styles (e.g., flexbox).
- `ui-qr-code`: The component is not widely used across projects or by the target audience, making it unnecessary to maintain within the SDS.
- `ui-social-follow-links`: The component adds unnecessary complexity to the library, both in terms of usage and maintenance.
- `ui-social-share-buttons`: Similar to `ui-social-follow-links`, this component adds unnecessary complexity to the library.
- `ui-teaser-article`: The functionality provided by the component is already covered by `sd-teaser` and `sd-media-teaser`.
- `ui-surrounding-map`: This component was heavily dependent on the Google Maps API and had often changing requirements. The SDS provides the `sd-map-marker` to be used inside a Google Maps map and in addition to this a guide on how to customize a Google Maps implementation by using the [Google Maps Style Wizard](https://mapstyle.withgoogle.com/).
```
